### PR TITLE
[4] deprecated iframe attributes

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue #32598
Replaces https://github.com/joomla/joomla-cms/pull/32666 which was incomplete and 15 days without improvements 
// cc @megh05

### Summary of Changes

Both the scrolling and frameborder are deprecated attributes, so I'd suggest removing these lines of code and also removing any associated parameters for them

### Testing Instructions

Create some wrapper iframe and check the output HTML for the iframe tags

### Actual result BEFORE applying this Pull Request

use of `scrolling` and `frameborder` deprecated tags, and associated params

### Expected result AFTER applying this Pull Request

no use of `scrolling` and `frameborder` deprecated tags, and associated params

### Documentation Changes Required

none

### CODE REVIEW

File: modules/mod_wrapper/mod_wrapper.xml [hunk 1]
```diff
@@ -83,18 +83,6 @@
 					<option value="1">JYES</option>
 				</field>
 
-				<field
```
Comment (Quy) [inline modules/mod_wrapper/mod_wrapper.xml:86]: Please delete lines 46-57.

Comment (PhilETaylor) [reply modules/mod_wrapper/mod_wrapper.xml:86]: I swear I did that.. I'll check again thanks.

Comment (Quy) [reply modules/mod_wrapper/mod_wrapper.xml:86]: Also please delete language strings. Thanks!

File: administrator/language/en-GB/com_wrapper.ini [hunk 2]
```diff
@@ -6,12 +6,10 @@
...
 COM_WRAPPER_FIELD_ADD_DESC="By default, http:// will be added unless it detects http:// or https:// in the URL you provide. This allows you to switch off this functionality."
 COM_WRAPPER_FIELD_ADD_LABEL="Auto Add"
-COM_WRAPPER_FIELD_FRAME_LABEL="Frame Border"
 COM_WRAPPER_FIELD_HEIGHT_LABEL="Height"
 COM_WRAPPER_FIELD_HEIGHTAUTO_LABEL="Auto Height"
 COM_WRAPPER_FIELD_LABEL_SCROLLBARSPARAMS="Scroll Bar Parameters"
 COM_WRAPPER_FIELD_LAZYLOADING_LABEL="Lazy Loading"
-COM_WRAPPER_FIELD_SCROLLBARS_LABEL="Scroll Bars"
 COM_WRAPPER_FIELD_URL_LABEL="URL"
 COM_WRAPPER_FIELD_VALUE_AUTO="Auto"
```
Comment (Quy) [inline administrator/language/en-GB/com_wrapper.ini]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e59dbdcce23e52d758a979b730142ce5cba9b276<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32913">issues.joomla.org/tracker/joomla-cms/32913</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on e59dbdcce23e52d758a979b730142ce5cba9b276<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32913">issues.joomla.org/tracker/joomla-cms/32913</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32913">issues.joomla.org/tracker/joomla-cms/32913</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**